tensorboard/plugins/histogram/tf_histogram_dashboard/tf-histogram-dashboard.html

```diff
@@ -163,6 +158,13 @@ <h3>No histogram data was found.</h3>
           type: Object,
           value: () => new tf_backend.RequestManager(),
         },
+
+        _getItemKey: {
+          type: Function,
+          value: function() {
+            return item => item.run + '/' + item.tag;
```

Comment:
Strictly speaking, this isn't a unique key, because we could have a run with slashes in it.  E.g. there could be a run `run1` with tag `eval/loss`, and a run `run1/eval` with a tag `loss`.

Perhaps better to do the JSON stringification approach here.